package.json

```diff
@@ -9,7 +9,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "eslint './**/*.{jsx,jsx,ts,tsx}'",
```

Comment:
Shouldn't it be like this? => eslint './**/*.{js,jsx,ts,tsx} 
.jsx extension file is duplicated